package.json

```diff
@@ -45,6 +45,7 @@
     "chai": "^4.1.2",
     "chromedriver": "^2.31.0",
     "glob": "^7.1.1",
+    "ip": "^1.1.5",
```

Comment:
For those curious. This package is described as:

>IP address utilities for node.js

 https://github.com/indutny/node-ip

Reply:
There may be a better solution than this.. but being we have to run visual tests within a container, this is used to find the IP address of the host machine so the headless browser in the container can access it.